automatic/lightscreen/tools/chocolateyInstall.ps1

```diff
@@ -8,8 +8,6 @@ $packageArgs = @{
   silentArgs   = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LAUNCHAFTER=0"
 }
 
-Start-Process 'AutoHotkey' "$toolsDir\install.ahk"
-
 Install-ChocolateyInstallPackage @packageArgs
 
 ps Lightscreen -ea 0 | kill
```

Comment:
Process might not start so fast on slower machines. Perhaps something like this may be more stable:

```powershell
for ($i=0; $i -lt 3; $i++) { sleep 1; $p = ps LightScreen -ea 0; if ($p) { $p | kill; Write-Host "Process killed:" $p.Name; break }  }
```

Reply:
Wow, thanks.